### ISSUE TITLE
Economy transfers to another world when it is set not to.

### ISSUE THREAD

**Issue by CaptainVSGames** [Gnat008/perworldinventory-kt#46]
So my problem is that when moving to another world that is not in the same group, the economy transfers over when I set it to save and load economy. Here are my settings.

config.yml: https://hastebin.com/valiwukuye.coffeescript
worlds.yml: https://hastebin.com/tazegocidu.php
DEBUG: https://hastebin.com/iyagosivex.vbs

Inventories transfer over perfectly fine.

**Comment by Gnat008** [comment]
Silly question, you do have Vault installed, right? If so can you post a startup log so I can see if it's being hooked correctly?

**Comment by CaptainVSGames** [comment]
@Gnat008 Yes, I do have vault. So sorry I didn't respond, was having personal problems.

FULL Startup Log: https://hastebin.com/wanawubiyi.md
PWI Hooking into Vault: https://hastebin.com/egirotokem.css

Also I changed my worlds.yml since last time: https://hastebin.com/ahudaxateb.php

**Comment by Gnat008** [comment]
Can you enable debugging, make the issue happen, and post that log? Should tell us more.
